src/Analyzers.UnitTests/Microsoft.Build.Analyzers.UnitTests.csproj

```diff
@@ -0,0 +1,92 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Shared\FileSystemSources.proj" />
+  <Import Project="..\Shared\DebuggingSources.proj" />
+
+  <PropertyGroup>
+    <!-- Don't build a netfx version of this unit with dotnet, as the netfx bootstrap layout created with 'dotnet build' is incomplete. -->
```

Comment:
#9767
It almost looks like we should not be trying to create a net472 bootstrap when building with .NET SDK because the required files are simply not there. For example, VS and SDK have diverged when it comes to SDK resolvers (`Microsoft.DotNet.MSBuildSdkResolver` vs `Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver`).